pixeltable/metadata/schema.py

```diff
@@ -179,6 +180,7 @@ class TableSchemaVersionMd:
     schema_version: int
     preceding_schema_version: Optional[int]
     columns: Dict[int, SchemaColumn]  # col_id -> SchemaColumn
+    table_attrs: TableAttributes
```

Comment:
The reason why TableParameters existed in the first place was that it was part of TableMd, but it's really just another needless level of indirection.

Why don't we correct that here and inline the attributes in TableSchemaVersionMd (because they really are part of a particular schema version) and ditch TableParameters/-Attributes altogether.